demo/index.html

```diff
@@ -106,7 +106,7 @@ <h1 class="c-main__title">React Components</h1>
               <a class="u-fg-inherit" href="chrome">Chrome</a>
             </div>
             <div class="u-mb-sm">
-              <a class="u-fg-inherit" href="dropdowns">Dropdowns</a>
+              <a class="u-fg-inherit" href="storybook">Dropdowns</a>
```

Comment:
Make this href navigate directly to the `dropdowns` section. This will also fix the current poor back button behavior. See `react-containers` for details.